docs/pages/Command-Line.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: default
+title: Command-Line
+permalink: /command-line/
+---
+
+- _TODO: update "Command-Line" document [#868](https://github.com/ChainSafe/gossamer/issues/868)_
+
+---
+
+## Gossamer Command
+
+The `gossamer` command is the root command for the `gossamer` package (`cmd/gossamer`). The root command starts the node (and initializes the node if the node has not already been initialized). 
+
+The `gossamer` command accepts the following ***local flags*** and ***global flags***:
```

Comment:
can you separate out local vs global flags? I see all the global flags being duplicated

Reply:
yes, see second bullet point in #868 this is how they are/were printed in the console

Reply:
we need to update the command-line first, see #857